File: Aula 03/exemplo1.py
# Exemplo: programa Python que utiliza uma função.
# A função somaDoisValores deve ser declarada antes de ser chamada.
def somaDoisValores(a,b):
    result = a + b
    return result

x = float(input("Digite o primeiro valor: "))
y = float(input("Digite o segundo valor: "))
print("A soma dos valores é: ", somaDoisValores(x, y))

File: Aula 07/Exemplo1.java
//Exemplo: programa Java que utiliza uma função.
import java.util.Scanner;
public class Exemplo1 {
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    System.out.println("Entre com o 1º valor");
    float x = sc.nextFloat();
    System.out.println("Entre com o 2º valor");
    float y = sc.nextFloat();
    System.out.println("A soma é " + somaDoisValores(x,y));
  }
  public static float somaDoisValores (float a, float b){
	  return a + b;
  }
}